shared/more/dumb-sheet.render.native.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+import React, {Component} from 'react'
+import {Box, Text, Input, Button} from '../common-adapters'
+import {globalStyles} from '../styles/style-guide'
+import type {DumbMap} from './dumb'
+
+import CommonMap from '../common-adapters/dumb.native'
+// import LoginMap from '../login/dumb.desktop'
```

Comment:
we'll pull this stuff over. should likely wait until all our outstanding prs are in